main.go

```diff
@@ -1,107 +1 @@
-package main
 
-import (
-	"database/sql"
-	"fmt"
-	"log"
-	"os"
-	
-
-	_ "github.com/mattn/go-sqlite3"
-)
-
-func openDB(dbfile string) (*sql.DB) {
-	db, err := sql.Open("sqlite3", dbfile)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return(db)
-}
-
-func getHandleID(db *sql.DB, handle string) ([]int){
-	var handleID []int
-
-	rows, err := db.Query("SELECT rowid, id, service FROM handle where id=?", handle )
-	// fmt.Println("SELECT rowid, id, service FROM handle where id='%s'", handle)
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer 	rows.Close()
-
-	for rows.Next() {
-		var rowid int
-		var id string
-		var service string
-		err = rows.Scan(&rowid, &id, &service)
-		if err != nil {
-			log.Fatal(err)
-		}
-		handleID = append(handleID, rowid)
-	}
-	err = rows.Err()
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return handleID
-}
-
-func getMessages(db *sql.DB, handles []int, name string) {
-	for _, handle := range handles {
-
-		query := `
-			select 
-			text, 
-			datetime(message.date/1000000000 + strftime("%s", "2001-01-01") ,"unixepoch","localtime"),
-			is_from_me 
-			from message 
-			WHERE handle_id=? 
-			ORDER by date
-		`
-		rows, err := db.Query(query, handle)
-		// fmt.Println("SELECT rowid, id, service FROM handle where id='%s'", handle)
-		if err != nil {
-			log.Fatal(err)
-		}
-		defer 	rows.Close()
-
-		for rows.Next() {
-			var text string
-			var date string
-			var is_from_me int
-			rows.Scan(&text, &date, &is_from_me)
-
-			if is_from_me == 1 {
-				fmt.Printf("%s,%s,%s\n", date, "Me", text)
-			} else {
```

Comment:
Typically we avoid `else` in go as it leads to deep-nesting which can be hard on the reader (unless you are intentionally coding like NASA https://www.reddit.com/r/programming/comments/aa3qdm/please_do_not_attempt_to_simplify_this_code_keep/).

Here is a good discussion of the style: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_return_early_rather_than_nesting_deeply

**Potential Exercise**: refactor this logic not to use else. Does the logic become more clear?